app/javascript/src/components/Dashboard/Notes/Item.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Dropdown, Avatar, Tooltip } from "neetoui";
+
+const Item = ({ note }) => (
+  <div className="m-3 rounded-sm border border-gray-200 p-4 text-gray-500 shadow-md">
+    <div className="flex justify-between">
+      <Typography style="h3">{note.title}</Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <li>Edit</li>
+        <li>Delete</li>
+      </Dropdown>
+    </div>
+    <Typography className="mb-3" style="body1">
+      {note.description}
+    </Typography>
+    <hr />
+    <div className="mt-4 flex justify-between">
+      <div className="note-tags">
+        {note.tags.map((tag, index) => (
+          <Tag
+            className="bg-gray-100 text-gray-500"
+            key={index}
```

Comment:
Please do not use the index as a key. [Refer](https://academy.bigbinary.com/learn-react/rendering-data-arrays/using-keys?searchTerms%5B0%5D=index#p-0-5:~:text=The%20default%20React,1). We expect you to not make these mistakes at this point of orientation.

Reply:
I understand. At least as far as the current scope of the assessment is concerned, you won't be needing this. However, make it a practice not to use `index` as keys. The idea is to prevent possible edge cases that may sprout up in the future. For example, if someone thinks of introducing a feature that enables you to delete the tags. 

The takeaway here is that using a unique key will give you a sense of assurance that you cannot get while using an `index` as the key.